sovtokenfees/sovtokenfees/test/dynamic_validation/test_nym_cannot_pay.py

```diff
@@ -0,0 +1,69 @@
+import json
+
+import pytest
+from sovtokenfees.test.helper import add_fees_request_with_address, ensure_all_nodes_have_same_data, \
+    get_amount_from_token_txn
+
+from plenum.common.exceptions import RequestRejectedException
+
+from plenum.test.helper import sdk_send_and_check, sdk_sign_request_from_dict
+
+from indy_common.authorize.auth_map import auth_map, add_new_identity_owner
+
+from indy_common.constants import NYM
+
+from indy_node.test.auth_rule.helper import sdk_send_and_check_auth_rule_request
+
+from indy_common.authorize.auth_actions import ADD_PREFIX
+
+
+def test_validation_nym_with_fees_cannot_pay(fees,
+                                             helpers,
+                                             nodeSetWithIntegratedTokenPlugin,
+                                             sdk_wallet_steward,
+                                             address_main,
+                                             sdk_pool_handle,
+                                             sdk_wallet_trustee,
+                                             mint_tokens,
+                                             looper):
+    """
+    Steps:
+    1. Checks that nym with fees will be rejected, because fees are not set
+    2. Send auth_rule txn with fees in metadata
```

Comment:
The description here is incorrect. The second NYM request is regecyed because it tries to pay more than the set Fees.